src/features/groups/components/ContainerCardList/ContainerCard/ContainerCardDropdownMenuTriggerButton/index.tsx

```diff
@@ -1,40 +1,63 @@
 import { IconMenuKebab } from '@/assets/images/icons';
-import { Button, DropdownMenu, DropdownMenuTrigger, Icon } from '@/components/ui';
+import { Button, DialogRoot, DropdownMenu, DropdownMenuTrigger, Icon } from '@/components/ui';
+import { useAutoFocus } from '@/hooks';
 import { IContainer } from '@/types/definition';
 
 import { useState } from 'react';
 
 import { ContainerCardDropdownMenuContent } from './ContainerCardDropdownMenuContent';
+import { DeleteContainerDialogContent } from './ContainerCardDropdownMenuContent/DeleteContainerDialogContent';
 
 interface IContainerCardDropdownMenuProps {
   /**
    * An identifier of a container which a user is willing to delete
    */
   containerId: IContainer['id'];
+  isRenameFormOpen: boolean;
```

Comment:
TSDoc for this prop. If it can be written in a single line, you can use this sysntax without a line break. Just a tip.
```ts
/** Write description here. */
isRenameFormOpen: boolean;
```